gradle/libs.versions.toml

```diff
@@ -6,6 +6,10 @@ managed-javadoc-parser = "0.3.1"
 managed-jsoup = "1.15.3"
 managed-jakarta-validation-api = "3.0.2"
 managed-slf4j = "1.7.36"
+managed-jspecify = "0.3.0"
```

Comment:
I don't think we should manage these dependencies. We should not publish them in the openAPI module's BOM.

Reply:
fixed